src/main/java/de/retest/recheck/cli/subcommands/Commit.java

```diff
@@ -115,7 +114,8 @@ private void applyChanges( final Persistence<SutState> persistence, final Review
 			try {
 				ApplyChangesToStatesFlow.apply( persistence, suiteChangeSet );
 			} catch ( final NoGoldenMasterFoundException e ) {
-				logger.error( "The Golden Master '{}' cannot be found.", e.getFilename() );
+				logger.error( "The following Golden Master(s) cannot be found:\n\t- {}",
```

Comment:
Couldn't you just log the exception's message, as this is already formatted in `NoGoldenMasterFoundException`?

Reply:
That would produce: 
```
Are you sure you want to continue? (y)es or (n)o:
No Golden Master file(s) with the following name(s) found:
	goldenMaster
Please make sure that the given test report '***' is within the corresponding project directory.
```

vs

```
Are you sure you want to continue? (y)es or (n)o:
The following Golden Master(s) cannot be found:
	- goldenMaster
Please make sure that the given test report '***' is within the corresponding project directory.
```

I think the current output (lower) is more understandable. Do we want to update this within recheck?

Reply:
I agree that the latter is better. But as you said, we probably want to have this in one place. Suggestion: Log the exception, let's fix the formatting in recheck directly later on?